epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -142,6 +155,43 @@ private boolean isFieldPackagePrivate(Element attribute) {
   }
 
   /**
+   * Checks if the given private field has getter and setter for access to it
+   */
+  private void findGetterAndSetterForPrivateField(ErrorLogger errorLogger) {
```

Comment:
The thing is that we can't enforce it since Kotlin generate automatically getters and setters for properties with this get/set naming pattern.